bazel/repository_locations.bzl

```diff
@@ -31,6 +31,13 @@ REPOSITORY_LOCATIONS = dict(
             "https://github.com/bazelbuild/rules_go/releases/download/v0.26.0/rules_go-v0.26.0.tar.gz",
         ],
     ),
+    io_bazel_rules_scala = dict(
+        sha256 = "76e1abb8a54f61ada974e6e9af689c59fd9f0518b49be6be7a631ce9fa45f236",
+        urls = [
+            "https://github.com/bazelbuild/rules_scala/archive/e7a948ad1948058a7a5ddfbd9d1629d6db839933.zip",
+        ],
+        strip_prefix = "rules_scala-%s" % "e7a948ad1948058a7a5ddfbd9d1629d6db839933",
```

Comment:
nit: Can't we do the following?

```
strip_prefix = "rules_scala-e7a948ad1948058a7a5ddfbd9d1629d6db839933"
```

Reply:
Updated. This was copy and pasted from when I had it in the WORKSPACE file and was taking advantage of using a variable.